PhIPSeq_external/Analysis_allergens_and_IEDB/predict_gender.py

```diff
@@ -0,0 +1,44 @@
+import string
+import seaborn as sns
+from matplotlib import pyplot as plt
+
+from PhIPSeq_external.Analysis_allergens_and_IEDB.prediction_helpers import get_oligos, read_metadata, configure_x_y, \
+    tune_hyper_parameters, plot_correlations, draw_auc, plot_shap
+
+
+def main(tune_params=False):
+    x = get_oligos()
+    meta = read_metadata()
+    meta = meta[['yob', 'gender']]
+    y = meta['gender']
+    meta.drop(columns=['gender'], inplace=True)
+    x, y = configure_x_y(x, y, meta, is_classifier=True)
```

Comment:
the order of these 3 lines is wierd (and why use drop).
Why not simply:
    y = meta['gender']
    meta = meta[['yob']]